bin/configs/springboot-petstore-server-spring-pageable.yaml

```diff
@@ -4,6 +4,6 @@ library: spring-boot
 inputSpec: modules/openapi-generator/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing-with-spring-pageable.yaml
 templateDir: modules/openapi-generator/src/main/resources/JavaSpring
 additionalProperties:
-  documentationProvider: springfox
+  documentationProvider: springdoc
```

Comment:
assuming `java8` removal seems to be similar to `bin/configs/springboot-petstore-server-spring-pageable-without-j8.yaml`